scripts/liminal

```diff
@@ -131,10 +131,6 @@ def deploy_liminal_apps(path, clean):
         yml_name = os.path.basename(config_file)
         target_yml_name = os.path.join(environment.get_dags_dir(), yml_name)
         shutil.copyfile(config_file, target_yml_name)
-        with open(config_file) as stream:
```

Comment:
won't this fail if config.load_kube_config() threw an exception?

Reply:
What do you mean? Here we don't load kube_config.
It takes place it liminal create.
And it won't fail in liminal create.
I cover it with try..catch